cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/setup/CheckImageProgress.java

```diff
@@ -0,0 +1,22 @@
+package com.sequenceiq.cloudbreak.cloud.event.setup;
+
+import com.sequenceiq.cloudbreak.domain.ImageStatus;
+
+public class CheckImageProgress {
+
```

Comment:
```
public static final int COMPLETED = 100;
public static final int INIT = 0;
```

can be added here and the ONE_HUNDRED, HALF etc. constans can be removed from everywhere.